environment/create_google_config.py

```diff
@@ -0,0 +1,308 @@
+def google_config_check():
+	
+	# Check for .google_cromwell.config in ~/
+	home = os.path.expanduser("~")
+	existance = os.path.exists(home + '/.google_cromwell.config')
+	
+	# If there is a configuration file, exit
+	if existance:
+		print "\nYou already have a Cromwell configuration file. If you would like to clear this setup\nand create a new file, remove (or rename) the hidden file ~/.google_cromwell.config\n"
+		sys.exit("Exiting.")
+
+# Ensure that gcloud SDK is installed, which is necessary to run the rest of the script. 
+def sdk_install_check():
+		
+	# If gcloud SDK is not installed, install it.
+	if os.system('gcloud version') is not 0:
+		
+		# Ask if user wants to install the SDK
+		installSdk = raw_input('\nStep (1): You do not have Google Cloud SDK installed, which you need to run this script.\nDo you want to install gcloud SDK? (yes or no)').lower()
+		
+		while not (installSdk.startswith("y") or installSdk.startswith("n")):
+			installSdk = raw_input('\nPlease answer yes or no: ').lower()
+			
+		# User chooses to install
+		if installSdk.startswith("y"):
+			os.system('curl https://sdk.cloud.google.com | bash')
+			shell = os.path.expanduser("$SHELL")
+			# Need to create new shell to start using gcloud
+			os.system('exec -l ' + shell)
+			os.system('gcloud init')
+			return
+		
+		# User chooses not to install SDK, exit the script because they can't continue.
+		elif installSdk.startswith("n"):
+			sys.exit("Exiting.")
+	
+	# Gcloud SDK is already installed, and user can continue with setup
+	else:
+		print "\nStep (1): You have Google Cloud SDK installed."
+		return
+
+# Which Google Project to use (new or existing)
+def which_google_project():
+
+	existing_project = raw_input('\nStep (2): Do you have an existing Google project where you want to run workflows? (yes or no) ').lower()
+
+	while not (existing_project.startswith("y") or existing_project.startswith("n")):	
+		existing_project = raw_input('\nPlease answer yes or no: ').lower()
+		
+	# User has existing project
+	if existing_project.startswith("y"):
+		global project_name
+		project_name = raw_input('\nEnter your Google project name: ')
+		create_google_bucket(project_name)
+		return project_name
+
+	# User doesn't have existing project
+	elif existing_project.startswith("n"):
+		print "\nIf you do not have a Google project you want to use, this script will generate a new one for you."
+		create_new_project = raw_input('\nWould you like to continue? (yes or no) ').lower()
+
+		while not (create_new_project.startswith("y") or create_new_project.startswith("n")):
+			create_new_project = raw_input('\nPlease answer yes or no: ').lower()
+
+		# Create new project
+		if create_new_project.startswith("y"):
+			find_billing_accounts()
+			# Which later creates the google project
+
+		# Don't create project, and exit
+		elif create_new_project.startswith("n"):
+			sys.exit("Exiting.")
```

Comment:
We should say why we are exiting here and maybe give people options or info about what to do next